src/js/src/components/SearchResult.vue

```diff
@@ -5,29 +5,62 @@
     </div>
     <div class="resultContainer">
       <h2>Results</h2>
-      <span v-if="!results.cardinality">
-        <span>Showing <span class="highlightText">{{ solrSettings.offset }}</span>  - <span class="highlightText">{{ solrSettings.offset + 20 > results.numFound ? results.numFound : solrSettings.offset + 20 }}</span> of </span>
-        <span class="highlightText">{{ results.numFound.toLocaleString("en") }}</span> entries matching <span class="highlightText">{{ query }}. </span>
-      </span>
-      <span v-if="results.cardinality">
-        <span>Showing <span class="highlightText">{{ solrSettings.offset }}</span> - <span class="highlightText">{{ solrSettings.offset + 20 > results.cardinality ? results.cardinality : solrSettings.offset + 20 }}</span> of </span>
-        <span class="highlightText">{{ results.cardinality.toLocaleString("en") }}</span> unique entries matching <span class="highlightText">{{ query }} </span>
-        <span class="tonedDownText">(total hits: {{ results.numFound.toLocaleString("en") }})</span>.
-      </span>
-      <div class="pagingContainer">
-        <button :disabled="solrSettings.offset < 20" @click="getPreviousResults()">
-          Previous 20
-        </button>
-        <button :disabled="results.cardinality ? solrSettings.offset + 20 > results.cardinality : solrSettings.offset + 20 > results.numFound" @click="getNextResults()">
-          Next 20
-        </button>
+      <!-- HERE COMES RESULTS // Figure out if this should be splitted out into a new component -->
```

Comment:
I agree

Reply:
Yes. I've done the splitting in the next commit.